scripts/mutect2_wdl/unsupported/mutect2_opt.wdl

```diff
@@ -274,10 +344,10 @@ task M2 {
   >>>
 
   runtime {
-      docker: "${m2_docker}"
-      memory: "5 GB"
-      disks: "local-disk " + 200 + " HDD"
-      preemptible: "${preemptible_attempts}"
+    docker: "${m2_docker}"
```

Comment:
This is a general WDL question, but why do some inputs in the runtime section need `${variable_name}` and others are just used as is (`variable_name`)?

Reply:
I'm changing all of these to just be the variable name where possible